geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geometry/Geometry.scala

```diff
@@ -11,9 +11,13 @@ package org.locationtech.geomesa.utils.geometry
 import com.vividsolutions.jts.geom.Polygon
 import org.geotools.geometry.jts.JTS
 
+@deprecated
 object Geometry {
+
+  @deprecated
   val noPolygon : Polygon  = null
 
+  @deprecated
```

Comment:
Can you add some details about the deprecation?  Is there something preferred in its place?

Reply:
no, i just noticed that we don't actually use it anywhere. do you think it's worth keeping around?